### PR TITLE
Fix ci

### LINKED ISSUE #105: Fix ci
Currently, dependencies installation is always failed in CI, since some complex requirements is lack of well tested installing process. I try to settle this problem by following the below solutions:
1.  Unify the dependency installation process in setup.sh.
2. Change the version of package scipy from 0.13.0b1 to 0.13.1

### PR DESCRIPTION
**Pull request by pkuyym**
resolves #105 
Fix ci following:
    1. Unify the dependency installation process in setup.sh.
    2. Change the version of package scipy from 0.13.0b1 to 0.13.1

### CODE REVIEW

File: deep_speech_2/setup.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# install python dependencies
+if [ -f 'requirements.txt' ]; then
+    pip install -r requirements.txt
+fi
+
```
Comment (xinghai-sun) [inline deep_speech_2/setup.sh]: remove the blank line

File: deep_speech_2/setup.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+fi
+tar -xvf libsamplerate-0.1.9.tar.gz
+cd libsamplerate-0.1.9
+./configure && make && make install
+cd -
+rm -rf libsamplerate-0.1.9
+rm libsamplerate-0.1.9.tar.gz
+pip install scikits.samplerate==0.3.3
+if [ $? != 0 ]; then
+    echo "Install sckikits.samplerate failed !!!"
```
Comment (xinghai-sun) [inline deep_speech_2/setup.sh]: sckikits --> scikits

File: deep_speech_2/setup.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    exit 1
+fi
```
Comment (xinghai-sun) [inline deep_speech_2/setup.sh]: If all successful, print something like "All done."

**Review by xinghai-sun — APPROVED** [review]
Almost LGTM.
Please also modify README.md accordingly.

File: deep_speech_2/setup.sh (hunk 1 above)
Comment (pkuyym) [reply deep_speech_2/setup.sh]: Done.

File: deep_speech_2/setup.sh (hunk 2 above)
Comment (pkuyym) [reply deep_speech_2/setup.sh]: Done.

File: deep_speech_2/setup.sh (hunk 3 above)
Comment (pkuyym) [reply deep_speech_2/setup.sh]: Done.

**Review by pkuyym — COMMENTED** [review]
Follow comments.

**Pull request merged**